include/clang/AST/ASTImporter.h

```diff
@@ -311,8 +311,10 @@ namespace clang {
     ///
     /// Subclasses can override this function to observe all of the \c From ->
     /// \c To declaration mappings as they are imported.
-    virtual Decl *Imported(Decl *From, Decl *To);
-      
+    virtual Decl *NotifyWhenImported(Decl *From, Decl *To) { return To; }
```

Comment:
Does it make sense to return not `To` in an override (this declaration suggests that there is a freedom to return something else)?

Reply:
Yes, you are right, I'll try to change the return value to `void`.